pkg/pillar/vendor/github.com/packetcap/go-pcap/pcap_linux.go

```diff
@@ -73,7 +75,7 @@ type Handle struct {
 }
 
 func (h *Handle) ReadPacketData() (data []byte, ci gopacket.CaptureInfo, err error) {
-	if !h.isOpen {
+	if h.isOpen > 0 {
```

Comment:
I think you should use the atomics to access this e.g., atomic.LoadUint32 to be consistent.

Reply:
I thought about it, but it wasn't necessary, since it checks later. But we can change it.